core/src/test/java/org/frankframework/parameters/BooleanParameterTest.java

```diff
@@ -0,0 +1,63 @@
+package org.frankframework.parameters;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.frankframework.core.PipeLineSession;
+import org.frankframework.stream.Message;
+import org.junit.jupiter.api.Test;
+
+public class BooleanParameterTest {
+
+	@Test
+	public void testBooleanParameterNumber() throws Exception {
+		BooleanParameter p = new BooleanParameter();
+		p.setName("number");
+		p.setValue("1");
+		p.configure();
+
+		ParameterValueList alreadyResolvedParameters = new ParameterValueList();
+		Message message = new Message("fakeMessage");
+
+		Object result = p.getValue(alreadyResolvedParameters, message, null, false);
+		assertInstanceOf(Boolean.class, result);
+		assertFalse((Boolean) result);
```

Comment:
`1`, of eigenlijk alle getallen ongelijk `0`, worden vaak als `true` geinterpreteerd.

Als `1` nu `false` wordt, zouden we niet eigenlijk beter dat een ongeldige waarde moeten maken voor een Boolean?

Reply:
Het is een ongeldige waarde, dus is het false. In properties kijken wij ook alleen naar true en niet naar 1 of !=0